app/models/order.rb

```diff
@@ -5,6 +5,9 @@ class Order < ApplicationRecord
   belongs_to :reward
   belongs_to :employee
 
+  scope :filtered_by_status, ->(status) { where(status: status) if status.present? }
+  scope :of_current_employee, ->(employee) { where(employee: employee) }
```

Comment:
perhaps since name is including _current_employee_ it should be something more like
```suggestion
  scope :of_current_employee, -> { where(employee: current_employee) }
```